examples/non-monorepo/turbo.json

```diff
@@ -0,0 +1,14 @@
+{
+  "$schema": "https://turbo.build/schema.json",
+  "pipeline": {
+    "build": {
+      "outputs": [".next/**"]
+    },
+    "lint": {
+      "outputs": []
```

Comment:
remind me again why this is necessary? I know it uses the defaults if you omit the `outputs` key, but is that harmful in some way?

Reply:
empty defaults array caches logs.

Reply:
We didn't change that behavior yet.